src/Compiler/Microsoft.AspNetCore.Razor.Language/src/CodeGeneration/CodeRenderingContext.cs

```diff
@@ -3,12 +3,13 @@
 
 #nullable disable
 
+using System;
 using System.Collections.Generic;
 using Microsoft.AspNetCore.Razor.Language.Intermediate;
 
 namespace Microsoft.AspNetCore.Razor.Language.CodeGeneration;
 
-public abstract class CodeRenderingContext
+public abstract class CodeRenderingContext : IDisposable
```

Comment:
New to put a `using` in both `TestCodeRenderingContext.cs` files for this change. 

Sorry, can't comment on code you didn't change so can't put this in the right place.

Reply:
The context is returned from those methods, and the usings are at the callsite of the Create method. Or are you talking about a different place in the file? Can you give me a line number?

Reply:
Oh, I think we're both right: seems I missed the callsites for `CreateRuntime()`, will update!